docs/contributing/set-up-dev-env.md

```diff
@@ -187,13 +187,48 @@ can take over 15 minutes to complete.
    hack/make.sh binary install-binary run
    ```
 
-9. Inside your container, check your Docker version.
+9. Inside your container, check your Docker versions:
 
    ```none
-   root@5f8630b873fe:/go/src/github.com/moby/moby# docker --version
-   Docker version 1.12.0-dev, build 6e728fb
+   # docker version
+   Client:
+    Version:      17.06.0-ce
+    API version:  1.30
+    Go version:   go1.8.3
+    Git commit:   02c1d87
+    Built:        Fri Jun 23 21:15:15 2017
+    OS/Arch:      linux/amd64
+
+   Server:
+    Version:      dev
+    API version:  1.35 (minimum version 1.12)
+    Go version:   go1.9.2
+    Git commit:   4aa6362da
+    Built:        Sat Dec  2 05:22:42 2017
+    OS/Arch:      linux/amd64
+    Experimental: false
    ```
+   
+   Notice the split versions between client and server, which might be unexpected. In more recent times the Docker CLI component (which provides the `docker` command) has split out from the `moby` project and is now maintained in [docker/docker-ce](https://github.com/docker/docker-ce).  The `moby` project now defaults to a [fixed version](https://github.com/docker/docker-ce/commits/v17.06.0-ce) of the `docker` CLI for integration tests.
```

Comment:
Can you

- wrap this paragraph to 80-chars?
- remove the back-ticks around `moby`, and use a capital (in this case it's describing the name of the project, not a command)

Reply:
Done and done.